connector/saml/saml.go

```diff
@@ -72,6 +79,17 @@ func init() {
 	}
 }
 
+// verifyBinding verifies binding from config
+func verifyBinding(binding string) (string, error) {
+	if binding == "" {
```

Comment:
nit: just set defaults in `openConnector`

Reply:
OK